.travis.yml

```diff
@@ -77,6 +77,7 @@ before_install:
   - pushd builds
   # Build into own virtualenv
   # We therefore control our own environment, avoid travis' numpy
+  - pip install -U 'virtualenv==14.0.4'
```

Comment:
Sure, not really sure I understand the problem. Something about `f2py` and the path. Maybe you could tell me what you would like to see here.